frontend/plugins/operation_ui/src/modules/cycle/components/CyclesColumns.tsx

```
@@ -1,5 +1,5 @@
 /* eslint-disable react-hooks/rules-of-hooks */
```

ESLint

[error] Definition for rule 'react-hooks/rules-of-hooks' was not found.

(react-hooks/rules-of-hooks)